--- apache-cassandra-4.1.3/conf/cassandra-env.sh.orig	2023-09-19 16:24:07.002998112 +0200
+++ apache-cassandra-4.1.3/conf/cassandra-env.sh	2023-09-19 16:24:38.427311610 +0200
@@ -88,7 +88,7 @@
 
 # Sets the path where logback and GC logs are written.
 if [ "x$CASSANDRA_LOG_DIR" = "x" ] ; then
-    CASSANDRA_LOG_DIR="$CASSANDRA_HOME/logs"
+    CASSANDRA_LOG_DIR="/var/log/cassandra"
 fi
 
 #GC log path has to be defined here because it needs to access CASSANDRA_HOME
