--- ./bin/cqlsh.patch_fix	2014-10-21 16:31:37.000000000 +0200
+++ ./bin/cqlsh	2015-01-12 23:25:27.805713525 +0100
@@ -74,7 +74,7 @@ CASSANDRA_PATH = os.path.join(os.path.di
 ZIPLIB_DIRS = [os.path.join(CASSANDRA_PATH, 'lib')]
 myplatform = platform.system()
 if myplatform == 'Linux':
-    ZIPLIB_DIRS.append('/usr/share/cassandra/lib')
+    ZIPLIB_DIRS.append('/usr/share/cassandra')
 
 if os.environ.get('CQLSH_NO_BUNDLED', ''):
     ZIPLIB_DIRS = ()
